pkg/kubernetes/announcement_handlers.go

```diff
@@ -0,0 +1,61 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openservicemesh/osm/pkg/announcements"
+	"github.com/openservicemesh/osm/pkg/constants"
+	"github.com/openservicemesh/osm/pkg/kubernetes/events"
+)
+
+// PatchSecretHandler patches the envoy bootstrap config secrets based on the PodAdd events
+func PatchSecretHandler(kubeClient kubernetes.Interface) {
```

Comment:
Would be good to add a unit test for this function.

Reply:
Unit test will be added in follow up PR